Makefile

```diff
@@ -177,19 +187,91 @@ scan:
 		--ignore-unfixed \
 		./
 
-print-%:
-	@echo -n $($*)
+.PHONY: build-utils-image
+build-utils-image: export IMAGE ?= $(EMBEDDED_OPERATOR_UTILS_IMAGE):$(EMBEDDED_OPERATOR_UTILS_IMAGE_VERSION)
+build-utils-image: export VERSION ?= $(EMBEDDED_OPERATOR_UTILS_IMAGE_VERSION)
+build-utils-image: export MELANGE_CONFIG = deploy/packages/utils/melange.tmpl.yaml
+build-utils-image: export APKO_CONFIG = deploy/images/utils/apko.tmpl.yaml
+build-utils-image: apko-build
 
-.PHONY: build-local-artifact-mirror-image
-build-local-artifact-mirror-image:
-	docker build --platform linux/amd64 -t $(LOCAL_ARTIFACT_MIRROR_IMAGE_LOCATION) -f deploy/local-artifact-mirror/Dockerfile .
+.PHONY: build-and-push-utils-image
+build-and-push-utils-image: export IMAGE ?= $(EMBEDDED_OPERATOR_UTILS_IMAGE):$(EMBEDDED_OPERATOR_UTILS_IMAGE_VERSION)
+build-and-push-utils-image: export VERSION ?= $(EMBEDDED_OPERATOR_UTILS_IMAGE_VERSION)
+build-and-push-utils-image: export APKO_CONFIG = deploy/images/utils/apko.tmpl.yaml
+build-and-push-utils-image: apko-login apko-build-and-publish
 
-.PHONY: push-local-artifact-mirror-image
-push-local-artifact-mirror-image:
-	docker push $(LOCAL_ARTIFACT_MIRROR_IMAGE_LOCATION)
+.PHONY: build-local-artifact-mirror-image
```

Comment:
ditto

Reply:
i did not add this but i did find it nice for local dev